### PR TITLE
Modified Messages

### PR DESCRIPTION
**Pull request by torgus**
Tested: yes.

@danielmayor

### CODE REVIEW

File: lang/en.inc.php [hunk 1]
```diff
@@ -37,19 +37,19 @@
 $messages['badcredentials'] = "Login or password incorrect";
 $messages['passworderror'] = "Password was refused by the LDAP directory";
 $messages['sshkeyerror'] = "SSH Key was refused by the LDAP directory";
-$messages['title'] = "Self service password";
+$messages['title'] = "Password Update";
```
Comment (magarciasopo) [inline lang/en.inc.php]: Updated

File: lang/es.inc.php [hunk 2]
```diff
@@ -76,8 +76,9 @@
 $messages['changehelpquestions'] = "<a href=\"?action=resetbyquestions\">Resetee su contraseña respondiendo preguntas</a>";
 $messages['changehelptoken'] = "<a href=\"?action=sendtoken\">Resetee su contraseña usando su e-mail</a>";
 $messages['changehelpsms'] = "<a href=\"?action=sendsms\">Resetee su contraseña mediante un SMS</a>";
-$messages['resetmessage'] = "Hola {login},\n\nClick aquí para restear su contraseña:\n{url}\n\n Si usted no es el emisor de esta petición, por favor ignórela.";
-$messages['resetsubject'] = "Reinicie su contraseña";
+$messages['resetmessage'] = "Estimado {login},\n\nClick aquí para resetear/cambiar su contraseña de LUCA (Smart Steps):\n{url}\n\n Si usted no es el emisor de esta petición, por favor ignórela. Saludos,
+Smart Steps Servicios de Ingeniera";
+$messages['resetsubject'] = "Change your LUCA (Smart Steps) pwd";
```
Comment (magarciasopo) [inline lang/es.inc.php]: pwd? password

File: lang/en.inc.php [hunk 3]
```diff
@@ -37,19 +37,19 @@
...
 $messages['submit'] = "Send";
 $messages['getuser'] = "Get user";
-$messages['tooshort'] = "Your password is too short";
-$messages['toobig'] = "Your password is too long";
-$messages['minlower'] = "Your password does not have enough lowercase characters";
-$messages['minupper'] = "Your password does not have enough uppercase characters";
-$messages['mindigit'] = "Your password does not have enough digits";
-$messages['minspecial'] = "Your password does not have enough special characters";
+$messages['tooshort'] = "Invalid password: the length must be at least 8 characters and contain at least one number and one alphanumeric character";
+$messages['toobig'] = "Invalid password: the length must be at least 8 characters and contain at least one number and one alphanumeric character";
```
Comment (danielmayor) [inline lang/en.inc.php]: If **too big**, does **at least 8 characters** make sense?

File: lang/en.inc.php [hunk 4]
```diff
@@ -82,8 +82,9 @@
 $messages['changehelpsms'] = "<a href=\"?action=sendsms\">Reset your password with a SMS</a>";
 $messages['changehelpsshkey'] = "<a href=\"?action=changesshkey\">Change your SSH Key</a>";
 $messages['changesshkeyhelp'] = "Enter your password and new SSH key.";
-$messages['resetmessage'] = "Hello {login},\n\nClick here to reset your password:\n{url}\n\nIf you didn't request a password reset, please ignore this email.";
-$messages['resetsubject'] = "Reset your password";
+$messages['resetmessage'] = "Dear {login},\n\nPlease click here to reset/change your LUCA (Smart Steps) password:\n{url}\n\nIf you didn't request a password reset, please ignore this email. Best regards,
+Smart Steps Engineering";
+$messages['resetsubject'] = "Change your LUCA (Smart Steps) pwd";
```
Comment (danielmayor) [inline lang/en.inc.php]: **password** instead of **pwd**

File: lang/es.inc.php [hunk 5]
```diff
@@ -76,8 +76,9 @@
 $messages['changehelpquestions'] = "<a href=\"?action=resetbyquestions\">Resetee su contraseña respondiendo preguntas</a>";
 $messages['changehelptoken'] = "<a href=\"?action=sendtoken\">Resetee su contraseña usando su e-mail</a>";
 $messages['changehelpsms'] = "<a href=\"?action=sendsms\">Resetee su contraseña mediante un SMS</a>";
-$messages['resetmessage'] = "Hola {login},\n\nClick aquí para restear su contraseña:\n{url}\n\n Si usted no es el emisor de esta petición, por favor ignórela.";
-$messages['resetsubject'] = "Reinicie su contraseña";
+$messages['resetmessage'] = "Estimado {login},\n\nClick aquí para resetear/cambiar su contraseña de LUCA (Smart Steps):\n{url}\n\n Si usted no es el emisor de esta petición, por favor ignórela. Saludos,
+Smart Steps Servicios de Ingeniera";
```
Comment (danielmayor) [inline lang/es.inc.php]: errata: **Ingeniería**

**Pull request merged**